mods/d2k/weapons.yaml

```diff
@@ -455,24 +455,21 @@ DeviatorMissile:
 	ReloadDelay: 160
 	Range: 5c0
 	Report: MISSLE1.WAV
-	InvalidTargets: Infantry, Structure
 	Projectile: Missile
 		MaximumLaunchSpeed: 281
-		RangeLimit: 40
```

Comment:
> That explains the missiles I've been seeing fly infinitely. Both from missile tanks and quads, I believe.

Quad rockets have a RangeLimit, although it might be too lenient.